CalibCalorimetry/HcalAlgos/src/HcalDbHardcode.cc

```diff
@@ -290,7 +290,7 @@ HcalRecoParam HcalDbHardcode::makeRecoParam (HcalGenericDetId fId) {
     //  param1.
     containmentCorrectionFlag = 1;         // p0
     containmentCorrectionPreSample = 0;    // p1
-    float phase  = 25.0;
+    float phase  = 6.0;
     float Xphase = (phase + 32.0) * 4.0;   // never change this line 
                                            //(offset 50nsec, 0.25ns step)
     Iphase       = Xphase;                 // p2
```

Comment:
Giovanni,

Iphase (at the end) is packed in the "param" (result) 
http://cmslxr.fnal.gov/source/CalibCalorimetry/HcalAlgos/src/HcalDbHardcode.cc#0236

former 25ns value was just wrong from the beginning, new value (6ns) 
corresponds to the Offline DB value. It did fix M0 scale in case 
_hardcoded_ parameters are used (which is the case for 2017 simulations at 
the moment).


On Wed, 24 Aug 2016, Giovanni Franzoni wrote:

> 
> In CalibCalorimetry/HcalAlgos/src/HcalDbHardcode.cc:
> 
> > @@ -290,7 +290,7 @@ HcalRecoParam HcalDbHardcode::makeRecoParam (HcalGener
> icDetId fId) {
> >      //  param1.
> >      containmentCorrectionFlag = 1;         // p0
> >      containmentCorrectionPreSample = 0;    // p1
> > -    float phase  = 25.0;
> > +    float phase  = 6.0;
> >      float Xphase = (phase + 32.0) * 4.0;   // never change this line 
> >                                             //(offset 50nsec, 0.25ns step)
> >      Iphase       = Xphase;                 // p2
> 
> hello @mariadalfonso
> 
> what is the change from 25 to 6 ascribed to?
> 
> At any rate, the chance 25 -> 6 propagates through:
> 
> phase => Xphase => Iphase
> 
> which are all local variables to makeRecoParam,
> it has no consequences.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the
> thread.[AEx02vXYQ4m9fJHV5bDVDtoNjJZdT0BSks5qjDB_gaJpZM4Jmhpf.gif]
> 
> 
>